Classes/GlobalStateExplorers/DatabaseBrowser/FLEXSQLiteDatabaseManager.m

```diff
@@ -114,9 +114,7 @@ - (NSInteger)lastRowID {
 }
 
 - (NSArray<NSArray *> *)queryAllDataInTable:(NSString *)tableName {
-    return [self executeStatement:[@"SELECT * FROM "
-        stringByAppendingString:tableName
-    ]].rows ?: @[];
+    return [self executeStatement:[NSString stringWithFormat:@"SELECT * FROM \"%@\"", tableName]].rows ?: @[];
```

Comment:
```suggestion
    NSString *command = [NSString stringWithFormat:@"SELECT * FROM \"%@\"", tableName];
    return [self executeStatement:command].rows ?: @[];
```

Reply:
Gotcha. That was my other interpretation :)